lib/recommendation.rb

```diff
@@ -0,0 +1,79 @@
+# suggestion for optimising: only pass in property id rather than full object?
```

Comment:
You should switch your code editor's default spacing to 2 spaces and convert all tabs to spaces.